ui/packages/components/src/Menu/MenuItem.tsx

```diff
@@ -32,12 +37,23 @@ export const MenuItem = ({
             comingSoon
               ? 'text-disabled hover:bg-disabled cursor-not-allowed'
               : active
-              ? 'bg-secondary-4xSubtle text-info hover:bg-secondary-3xSubtle'
+              ? 'bg-secondary-3xSubtle text-info hover:bg-secondary-2xSubtle'
               : 'hover:bg-canvasSubtle text-muted'
```

Comment:
The design system has the hover state also make the text brighter. I think it needs something like this:
```suggestion
              : 'hover:bg-canvasSubtle text-muted hover:text-basis'
```